docs/developers/completable-future.md

```diff
@@ -0,0 +1,78 @@
+
+---
+title: Using CompletableFuture
+---
+
+Velocity makes use of *CompletableFuture* in many places. But what does that mean? How do I handle a future? Why even use them? You're about to find out.
```

Comment:
can we add "makes use of Java's *CompletableFuture*"?

Reply:
I thought that would be redundant, since it is mentioned in the next paragraph.

Reply:
It makes more sense to have it mentioned in the beginning to establish where it is from, then explain a little bit about it afterwards